save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CreationView.kt

```diff
@@ -352,15 +359,42 @@ class CreationView : AbstractView<Props, ProjectSaveViewState>(true) {
 
-                                        div("form-check form-switch mt-2") {
-                                            input(classes = "form-check-input") {
-                                                attrs["type"] = "checkbox"
-                                                attrs["id"] = "isPublicSwitch"
-                                                attrs["checked"] = "true"
+                                        div("col-md-12 mt-3 mb-3 pl-2 pr-0 row") {
+                                            div("col-5 text-left align-self-center") {
+                                                +"Project visibility:"
                                             }
-                                            label("form-check-label") {
-                                                attrs["htmlFor"] = "isPublicSwitch"
-                                                +"Public project"
+                                            form("col-7 form-group row d-flex justify-content-around") {
+                                                div("form-check-inline") {
+                                                    input(classes = "form-check-input") {
+                                                        attrs.defaultChecked = state.isPublic!!
+                                                        attrs["name"] = "projectVisibility"
+                                                        attrs["type"] = "radio"
+                                                        attrs["id"] = "isProjectPublicSwitch"
+                                                        attrs["value"] = "true"
+                                                    }
+                                                    label("form-check-label") {
+                                                        attrs["htmlFor"] = "isProjectPublicSwitch"
+                                                        +"Public"
+                                                    }
+                                                }
+                                                div("form-check-inline") {
+                                                    input(classes = "form-check-input") {
+                                                        attrs.defaultChecked = !state.isPublic!!
+                                                        attrs["name"] = "projectVisibility"
+                                                        attrs["type"] = "radio"
+                                                        attrs["id"] = "isProjectPrivateSwitch"
+                                                        attrs["value"] = "false"
+                                                    }
+                                                    label("form-check-label") {
+                                                        attrs["htmlFor"] = "isProjectPrivateSwitch"
+                                                        +"Private"
```

Comment:
It would be useful to also have a popup here, which explains publicity: private projects are visible for user, organization admins and selected users, while public ones are visible for everyone.

Reply:
Yes, I agree. Added